Jenkinsfile

```diff
@@ -89,11 +89,11 @@ pipeline {
                 withCredentials([usernamePassword(credentialsId: 'artifactory-gooey', \
                                                     usernameVariable: 'artifactoryUser', \
                                                     passwordVariable: 'artifactoryPass')]) {
-                    sh '''./gradlew
-                        --console=plain
-                        -Dorg.gradle.internal.publish.checksums.insecure=true
-                        publish
-                        -PmavenUser=${artifactoryUser}
+                    sh '''./gradlew \\
```

Comment:
do we really need two? 🤔 
also I noticed, we probably need double-quotes to allow variable substitution for the creds.